.eslintrc.json

```diff
@@ -47,7 +47,8 @@
     "semi": "off",
     "@typescript-eslint/semi": ["error", "never"],
     "@typescript-eslint/type-annotation-spacing": "error",
-    "@typescript-eslint/unbound-method": "error"
+    "@typescript-eslint/unbound-method": "error",
+    "sort-imports": "off"
```

Comment:
Why? You don't like it? Or was there an issue?

Reply:
I could not pass the CI, I can't order the imports in a way that suits all rules "sort-import", "multiple before single" etc :cry: 
From someone here who does java-script every days, I got told we don't care very much about this, it does not change a thing so I followed the advice

Reply:
Ok fine by me, let's go 🚀